components/src/molecules/tabs/Tabs.scss

```diff
@@ -36,12 +37,21 @@ $tabs__content-padding\@large   : $spacer--large $spacer--extra-large !default;
       background-color: $tabs__button-background--active;
       border-bottom: $tabs__button-border--active;
     }
+
+    &:focus {
+      z-index: 1;
+      outline: $tabs__outline-focus;
+    }
```

Comment:
Can we show focus only for keyboard users? Lately, I talked with Ania there is some solution. As I remember it works for firefox but there is also a polyfill. Please ask Ania about details. If it works we should add it globally not only for tabs.